public/js/components/SourceFooter.css

```diff
@@ -1,3 +1,10 @@
+.source-footer {
+  border-top: 1px solid var(--theme-splitter-color);
+  left: 0;
+  opacity: 1.0;
```

Comment:
Looks like the lint doesn't like the trailing zero here.

```
public/js/components/SourceFooter.css
 4:13  ✖  Unexpected trailing zero(s)   number-no-trailing-zeros
```

Otherwise good to go.